packages/vue/src/components/organisms/SfMegaMenu/SfMegaMenu.stories.js

```diff
@@ -1,8 +1,295 @@
 
-storiesOf("Organisms|MegaMenu", module).add("Common", () => ({
-  components: { SfMegaMenu },
-  template: "<SfMegaMenu>Hello Mega-Menu</SfMegaMenu>"
-}));
+const AsidePlaceholder = {
+  components: { SfImage },
+  data() {
+    return {
+      tiles: [
+        {
+          title: "Last pairs left",
+          pictures: {
+            small: {
+              url: "assets/storybook/megamenu/bannerSandals-full.png"
+            },
+            normal: { url: "assets/storybook/megamenu/bannerSandals.jpg" }
+          }
+        },
+        {
+          title: "Beach bags 2=1",
+          pictures: {
+            small: {
+              url: "assets/storybook/megamenu/bannerBeachBag-full.png"
+            },
+            normal: { url: "assets/storybook/megamenu/bannerBeachBag.jpg" }
+          }
+        }
+      ]
+    };
+  },
+  template: `<div :style="{display: 'flex', justifyContent: 'space-between', flexWrap: 'wrap'}">
+      <div 
+        v-for="tile in tiles" 
+        :key="tile.title" 
+        :style="{marginBottom: '1.25rem'}"
+      >
+        <h3 :style="{marginBottom: '1.25rem', textTransform: 'uppercase'}">{{tile.title}}</h3>
+        <SfImage :src="tile.pictures"/>
+      </div>
+    </div>`
+};
+const MegaMenuPlaceholder = {
+  components: { SfMegaMenu, SfMenuItem, AsidePlaceholder },
+  props: {
+    title: {
+      type: String,
+      default: ""
+    },
+    visible: {
+      type: Boolean,
+      default: false
+    }
+  },
+  data() {
+    return {
+      visible: true,
+      subcategories: [
+        {
+          header: "Clothing",
+          items: [
+            { label: "Skirts" },
+            { label: "Sweaters" },
+            { label: "Dresses" },
+            { label: "TShirts" },
+            { label: "Pants" },
+            { label: "Underwear" },
+            { label: "Jackets" },
+            { label: "Blouses" }
+          ]
+        },
+        {
+          header: "Accessories",
+          items: [
+            { label: "Bags & Purses" },
+            { label: "Belts" },
+            { label: "Gloves" },
+            { label: "Hats" }
+          ]
+        },
+        {
+          header: "Shoes",
+          items: [
+            { label: "Boots" },
+            { label: "Heels" },
+            { label: "Flat shoes" },
+            { label: "Loafers" },
+            { label: "Sandals" },
+            { label: "Slippers" },
+            { label: "Trainers" }
+          ]
+        }
+      ]
+    };
+  },
+  template: `<SfMegaMenu 
+        title="Man"
+        :visible="visible"
+        :style="{ position: 'absolute', left: 0, width: '100%' }"
+      >
+        <SfMegaMenuColumn 
+          v-for="subcategory in subcategories" 
+          :key="subcategory.header" 
+          :title="subcategory.header"
+        >
+          <SfMenuItem 
+            v-for="item in subcategory.items" 
+            :key="item.label" 
+            :label="item.label" 
+            class="sf-mega-menu__menu-item"
```

Comment:
why do we need this class?

Reply:
just for adjust to the design, we can add SfMegaMenuItem and add this class inside
![_1](https://user-images.githubusercontent.com/12138170/71894470-a09bda00-314e-11ea-9def-99b48c2e1498.png)
![_2](https://user-images.githubusercontent.com/12138170/71894471-a09bda00-314e-11ea-9510-1db739c9c481.png)

Reply:
@filrak I added SfMegaMenuItem to wrap SfMnuItem and add class